src/utils/__tests__/style-formatter.spec.js

```diff
@@ -1,26 +1,13 @@
-import styleFormatter from '../utils/style-formatter';
+import styleFormatter from '../style-formatter';
+import defaultValues from '../../components/__tests__/default-button-props';
```

Comment:
@niekvanstaveren I don't like rewriting code, but this import might be questionable 🙃